app/Http/Controllers/Api/ApiController.php

```diff
@@ -64,13 +64,17 @@ public function item($data, $code = 200, $meta = [], $transformer = null)
      * @param  array  $meta
      * @return \Illuminate\Http\JsonResponse
      */
-    public function transform($data, $code = 200, $meta = [])
+    public function transform($data, $code = 200, $meta = [], $include = null)
```

Comment:
@sbsmith86 okay cool it seems like I am the most bothered by it. I thought about that too, but there are two functions that pass in `meta` and only this one passes in `includes`.